content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -73,567 +79,669 @@ Electronic Invoice Credentials
 
-.. image:: argentina/argentina04.png
+.. image:: argentina/identification-types.png
    :align: center
+   :alt: A list of AR Localization document types in Odoo, as defined by AFIP.
 
 .. note::
-   The complete list of Identification types defined by the AFIP is included in Odoo but only the
-   common ones are active.
+   The complete list of :guilabel:`Identification Types` defined by the AFIP is included in Odoo,
+   but only the common ones are active.
 
-AFIP Responsibility Type
+AFIP responsibility type
 ************************
 
-In Argentina the document type associated with customers and vendors transactions is defined based
-on the AFIP Responsibility type, this field should be defined in the partner form:
+In Argentina, the document type and corresponding transactions associated with customers and
+vendors is defined by the AFIP Responsibility type. This field should be defined in the **Partner
+form**.
 
-.. image:: argentina/argentina05.png
+.. image:: argentina/select-afip-responsibility-type.png
    :align: center
+   :alt: Select AFIP Responsibility Type.
 
 Taxes
 ~~~~~
 
-As part of the localization module,  the taxes are created automatically with their related
-financial account and configuration.
+As part of the localization module, the taxes are created automatically with their related
+financial account and configuration, e.g., 73 taxes for :guilabel:`Responsable Inscripto`.
 
-.. image:: argentina/argentina06.png
+.. image:: argentina/automatic-tax-configuration.png
    :align: center
+   :alt: A list of AR Localization taxes with financial amount and configuration in Odoo.
 
-Taxes Types
+Taxes types
 ***********
 
 Argentina has several tax types, the most common ones are:
 
-- VAT. Is the regular VAT and it can have several percentages.
-- Perception. Advance payment of a tax that is applied on Invoices.
-- Retention. Advance payment of a tax that is applied on payments
-- Otros.
+- :guilabel:`VAT`: this is the regular VAT and can have various percentages;
+- :guilabel:`Perception`: advance payment of a tax that is applied on invoices;
+- :guilabel:`Retention`: advance payment of a tax that is applied on payments.
 
-Special Taxes
+Special taxes
 *************
 
-Some argentine taxes are not commonly used  for all companies, these type of taxes are included as
-inactive by default, it's important that before creating a new tax you confirm if they are not
-already included in the Inactive taxes:
+Some Argentinean taxes are not commonly used for all companies, and those less common options are
+labeled as inactive in Odoo by default. Before creating a new tax, be sure to check if that tax is
+not already included as inactive.
 
-.. image:: argentina/argentina07.png
+.. image:: argentina/special-inactive-taxes.png
    :align: center
+   :alt: A list showing less common Argentinean tax options, which are labeled as inactive in Odoo
+         by default.
+
+.. _document-types:
```

Comment:
This would conflict with any other `document-types` anchor added later in the doc, which is likely to happen.
```suggestion
.. _localizations/argentina/document-types:
```